### DISCUSSION TITLE
Right Drawer Rendering

### DISCUSSION THREAD

**Discussion by KWangechi** [filamentphp/filament#13308] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

Hey everyone, anyone know how to render a right drawer or a left drawer? I am trying to have it rendered when I click on a row in the table instead of having the default behavior of opening the edit page...

Will really appreciate

**Comment by leandrocfe** [comment]
Right drawer? Do you mean, `slideOver()`?
https://filamentphp.com/docs/3.x/actions/modals#using-a-slide-over-instead-of-a-modal

**Reply by KWangechi** [reply]
Thanks, I got it. I wanted to use it in a blade file (I was using Alpine JS with some external JavaScript code)
